crates/iota-sdk-ffi/src/types/transaction/mod.rs

```diff
@@ -1532,12 +1518,6 @@ impl Argument {
             subresult_index,
         ))
     }
-
-    /// Turn a Result into a NestedResult. If the argument is not a Result,
-    /// returns None.
-    pub fn nested(&self, ix: u16) -> Option<Arc<Argument>> {
-        self.0.nested(ix).map(Self).map(Arc::new)
-    }
```

Comment:
Why delete this?

Reply:
Ok found the reason below, it's replaced by the other methods like `as_nested_result_opt`

Reply:
Yeah but this method is kind of weird and also poorly named. I also don't see the value in it

Reply:
I re-added it but called it `get_nested`